custom-write-strategy-example/README.md

```diff
@@ -0,0 +1,25 @@
+# MongoDB Kafka connector customization
+
+Example of writing a custom write strategy
+
+./gradlew clean build
+
+Then copy the `mongo-kafka-customization/build/libs/UpsertAsPartOfDocumentStrategy.jar`
```

Comment:
nit: Update the directory name to `./build/libs/UpsertAsPartOfDocumentStrategy.jar`